pkg/util/derived/derived.go

```diff
@@ -30,18 +33,33 @@ func baseCloudProviderConf(cs *api.OpenShiftManagedCluster, cloudProviderBackoff
 		cfg.CloudProviderRateLimitQPS = 3.0
 		cfg.CloudProviderRateLimitBucket = 10
 	}
-	return &cfg
+	// match any version v8+
+	// TODO: make this default when all clusters are v8+
+	version, err := regexp.Compile("^v([8-9]|\\d{2,}).*")
```

Comment:
Hrm I am not a fan of doing regex semver matching, and this introduces a new exciting way to feature flag config changes between earlier versions... can you just add another bool like cloudProviderBackoff that we can purge once the code that uses it is gone?